### PR TITLE
DESCW-1879 Feature/alpha theme addition

### PR DESCRIPTION
**Pull request by codewisenate**
- Added Alpha v3 header and footer including alpha.json theme styles using BC Government design system tokens;
- Includes various touch points such as: alpha.json styles using Alpha v3 design system tokens; 
- theme.json addition of header/footer and search bar templates; 
- New default logo asset for alpha style header; 
- Various and extensive styles to drive new templating options; 
- Search specific Javascript to drive toggle interaction;
- WordPress hooks to augment: 
-- default site logo for Alpha design, 
-- body class to enable Alpha specific styling, 
-- additional injected styles for mobile breakpoint menu styling.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 1.4.1 Jan 24, 2024
```
Comment (ShawnTurple) [inline CHANGELOG.md]:
```suggestion
## 1.5.0 Jan 24, 2024
```

File: composer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "bcgov-theme/bcgov-wordpress-block-theme",
     "description": "Block Theme for BC Gov",
-    "version": "1.4.0",
+    "version": "1.4.1",
```
Comment (ShawnTurple) [inline composer.json]:
```suggestion
    "version": "1.5.0",
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bcgov-wordpress-block-theme",
-  "version": "1.4.0",
+  "version": "1.4.1",
```
Comment (ShawnTurple) [inline package.json]:
```suggestion
  "version": "1.5.0",
```

File: parts/footer-alpha-v3.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+<div class="wp-block-group has-white-color has-nonary-background-color has-text-color has-background" style="border-top-color:var(--wp--preset--color--secondary-brand);border-top-width:4px;border-right-style:none;border-right-width:0px;border-bottom-color:var(--wp--preset--color--secondary-brand);border-bottom-width:4px;border-left-style:none;border-left-width:0px;padding-top:2rem;padding-right:2rem;padding-bottom:2rem;padding-left:2rem"><!-- wp:paragraph {"style":{"typography":{"fontSize":"1rem"}}} -->
+	<p style="font-size:1rem">The BC Public Service acknowledges the territories of First Nations around B.C. and is grateful to carry out our work on these lands. We acknowledge the rights, interests, priorities and concerns of all Indigenous Peoples—First Nations, Métis and Inuit—respecting and acknowledging their distinct cultures, histories, rights, laws and governments.</p>
```
Comment (ShawnTurple) [inline parts/footer-alpha-v3.html:3]: just curious where this text came from, and why it should be the default (i like it, just want to know the logic behind it)

File: src/scripts/public/index.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import '@bcgov/design-tokens/css-prefixed/variables.css';
```
Comment (ShawnTurple) [inline src/scripts/public/index.js:11]: so this is cool that you are using the design tokens, now how do we tell our future selfs, that these tokens are in the styles, i almost feel like a header comment block above the declaration would be helpful, to indicate their use and what they look like ie `var(--bcds-surface-brand-gray-30)`

Comment (ShawnTurple) [reply src/scripts/public/index.js:11]: for this matter, we might want to do this for all of our var(--wp-sample-color), to indicate what are wordpress out of the box, and what ones belong to the theme.  Setting a standard of sorts

File: src/styles/public/index.scss [hunk 6]
```diff
@@ -36,3 +40,7 @@
 
 // Print Styles
 @import "./print";
+
+// body {
+// 	--wp--preset--color--primary-brand: var(--bcds-surface-brand-gold-60);
```
Comment (ShawnTurple) [inline src/styles/public/index.scss]: was this a test, should it still be here, if not please remove

File: style.css [hunk 7]
```diff
@@ -7,7 +7,7 @@ Description: WordPress Block theme is a theme that leverages the full-site editi
 Requires at least: 6.2
 Tested up to: 6.2.2
 Requires PHP: 7.4
-Version: 1.4.0
+Version: 1.4.1
```
Comment (ShawnTurple) [inline style.css]:
```suggestion
Version: 1.5.0
```

File: styles/alpha.json [hunk 8]
```diff
@@ -0,0 +1,228 @@
+{
+	"version": 2,
+	"title": "Alpha v3",
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"palette": [
+				{
+					"slug": "primary-brand",
+					"color": "var(--bcds-surface-primary-default)",
```
Comment (ShawnTurple) [inline styles/alpha.json:10]: nice that we can just use this variable

**Review by ShawnTurple — CHANGES_REQUESTED** [review]
* this is not a bug fix, therefor minor change, please change all to 1.5.0
* adding some additional comments with regards to design system tokens
* expected that vuepress documentation will be a separate PR

File: parts/footer-alpha-v3.html (hunk 4 above)
Comment (codewisenate) [reply parts/footer-alpha-v3.html:3]: This text came from the Alpha 3 update Figma attached to the ticket as part of the design system layout: https://www.figma.com/file/7qqTHUz3u9nYN17Tdwfvg3/Alpha-3.0-updates?node-id=1%3A6415&mode=dev

File: src/scripts/public/index.js (hunk 5 above)
Comment (codewisenate) [reply src/scripts/public/index.js:11]: I have added a JSDoc to the top of this and the admin/index.js to explain the Alpha styles and prefixed variables relationship.

File: styles/alpha.json (hunk 8 above)
Comment (codewisenate) [reply styles/alpha.json:10]: Agreed. For once WordPress didn't fight us on something that should be (and is) trivial ;)

**Review by ShawnTurple — DISMISSED** [review]

**Review by kshapka-bcgov — APPROVED** [review]

**Pull request merged**